# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Nuno Amorim <inovaty@gmail.com>, 2018
# Hugo Carvalho <hugokarvalho@hotmail.com>, 2019
# Ricardo Simões <xmcorporation@gmail.com>, 2019
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-06-18 15:54+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Ricardo Simões <xmcorporation@gmail.com>, 2019\n"
"Language-Team: Portuguese (Portugal) (https://www.transifex.com/calamares/teams/20061/pt_PT/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: pt_PT\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: src/modules/grubcfg/main.py:37
msgid "Configure GRUB."
msgstr "Configurar o GRUB."

#: src/modules/mount/main.py:38
msgid "Mounting partitions."
msgstr "A montar partições."

#: src/modules/mount/main.py:150 src/modules/initcpiocfg/main.py:187
#: src/modules/initcpiocfg/main.py:191
#: src/modules/luksopenswaphookcfg/main.py:95
#: src/modules/luksopenswaphookcfg/main.py:99 src/modules/rawfs/main.py:171
#: src/modules/machineid/main.py:49 src/modules/initramfscfg/main.py:94
#: src/modules/initramfscfg/main.py:98 src/modules/openrcdmcryptcfg/main.py:78
#: src/modules/openrcdmcryptcfg/main.py:82
#: src/modules/luksbootkeyfile/main.py:51 src/modules/fstab/main.py:312
#: src/modules/fstab/main.py:316 src/modules/localecfg/main.py:144
#: src/modules/networkcfg/main.py:48
msgid "Configuration Error"
msgstr "Erro de configuração"

#: src/modules/mount/main.py:151 src/modules/initcpiocfg/main.py:188
#: src/modules/luksopenswaphookcfg/main.py:96 src/modules/rawfs/main.py:172
#: src/modules/initramfscfg/main.py:95 src/modules/openrcdmcryptcfg/main.py:79
#: src/modules/luksbootkeyfile/main.py:52 src/modules/fstab/main.py:313
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "Nenhuma partição está definida para <pre>{!s}</pre> usar."

#: src/modules/services-systemd/main.py:35
msgid "Configure systemd services"
msgstr "Configurar serviços systemd"

#: src/modules/services-systemd/main.py:68
#: src/modules/services-openrc/main.py:102
msgid "Cannot modify service"
msgstr "Não é possível modificar serviço"

#: src/modules/services-systemd/main.py:69
msgid ""
"<code>systemctl {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"<code>systemctl {arg!s}</code> chamar pelo chroot retornou com código de "
"erro {num!s}."

#: src/modules/services-systemd/main.py:72
#: src/modules/services-systemd/main.py:76
msgid "Cannot enable systemd service <code>{name!s}</code>."
msgstr "Não é possível ativar o serviço systemd <code>{name!s}</code>."

#: src/modules/services-systemd/main.py:74
msgid "Cannot enable systemd target <code>{name!s}</code>."
msgstr "Não é possível ativar o destino do systemd <code>{name!s}</code>."

#: src/modules/services-systemd/main.py:78
msgid "Cannot disable systemd target <code>{name!s}</code>."
msgstr "Não é possível desativar o destino do systemd <code>{name!s}</code>."

#: src/modules/services-systemd/main.py:80
msgid "Cannot mask systemd unit <code>{name!s}</code>."
msgstr "Não é possível mascarar a unidade do systemd <code>{name!s}</code>."

#: src/modules/services-systemd/main.py:82
msgid ""
"Unknown systemd commands <code>{command!s}</code> and "
"<code>{suffix!s}</code> for unit {name!s}."
msgstr ""
"Comandos do systemd desconhecidos <code>{command!s}</code> e "
"<code>{suffix!s}</code> por unidade {name!s}."

#: src/modules/umount/main.py:40
msgid "Unmount file systems."
msgstr "Desmontar sistemas de ficheiro."

#: src/modules/unpackfs/main.py:41
msgid "Filling up filesystems."
msgstr "A preencher os sistemas de ficheiros."

#: src/modules/unpackfs/main.py:159
msgid "rsync failed with error code {}."
msgstr "rsync falhou com código de erro {}."

#: src/modules/unpackfs/main.py:220 src/modules/unpackfs/main.py:238
msgid "Failed to unpack image \"{}\""
msgstr "Falha ao descompactar imagem \"{}\""

#: src/modules/unpackfs/main.py:221
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed"
msgstr ""
"Falha ao procurar unsquashfs, certifique-se que tem o pacote squashfs-tools "
"instalado"

#: src/modules/unpackfs/main.py:320
msgid "No mount point for root partition"
msgstr "Nenhum ponto de montagem para a partição root"

#: src/modules/unpackfs/main.py:321
msgid "globalstorage does not contain a \"rootMountPoint\" key, doing nothing"
msgstr "globalstorage não contém um \"rootMountPoint\" chave, nada a fazer"

#: src/modules/unpackfs/main.py:326
msgid "Bad mount point for root partition"
msgstr "Ponto de montagem mau para partição root"

#: src/modules/unpackfs/main.py:327
msgid "rootMountPoint is \"{}\", which does not exist, doing nothing"
msgstr "rootMountPoint é \"{}\", que não existe, nada a fazer"

#: src/modules/unpackfs/main.py:340 src/modules/unpackfs/main.py:347
#: src/modules/unpackfs/main.py:352
msgid "Bad unsquash configuration"
msgstr "Má configuração unsquash"

#: src/modules/unpackfs/main.py:341
msgid "The filesystem for \"{}\" ({}) is not supported"
msgstr "O sistema de ficheiros \"{}\" ({}) não é suportado"

#: src/modules/unpackfs/main.py:348
msgid "The source filesystem \"{}\" does not exist"
msgstr "O sistema de ficheiros fonte \"{}\" não existe"

#: src/modules/unpackfs/main.py:353
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "O destino \"{}\" no sistema alvo não é uma diretoria"

#: src/modules/displaymanager/main.py:381
msgid "Cannot write KDM configuration file"
msgstr "Não é possível gravar o ficheiro de configuração KDM"

#: src/modules/displaymanager/main.py:382
msgid "KDM config file {!s} does not exist"
msgstr "O ficheiro de configuração do KDM {!s} não existe"

#: src/modules/displaymanager/main.py:443
msgid "Cannot write LXDM configuration file"
msgstr "Não é possível gravar o ficheiro de configuração LXDM"

#: src/modules/displaymanager/main.py:444
msgid "LXDM config file {!s} does not exist"
msgstr "O ficheiro de configuração do LXDM {!s} não existe"

#: src/modules/displaymanager/main.py:527
msgid "Cannot write LightDM configuration file"
msgstr "Não é possível gravar o ficheiro de configuração LightDM"

#: src/modules/displaymanager/main.py:528
msgid "LightDM config file {!s} does not exist"
msgstr "O ficheiro de configuração do LightDM {!s} não existe"

#: src/modules/displaymanager/main.py:602
msgid "Cannot configure LightDM"
msgstr "Não é possível configurar o LightDM"

#: src/modules/displaymanager/main.py:603
msgid "No LightDM greeter installed."
msgstr "Nenhum ecrã de boas-vindas LightDM instalado."

#: src/modules/displaymanager/main.py:634
msgid "Cannot write SLIM configuration file"
msgstr "Não é possível gravar o ficheiro de configuração SLIM"

#: src/modules/displaymanager/main.py:635
msgid "SLIM config file {!s} does not exist"
msgstr "O ficheiro de configuração do SLIM {!s} não existe"

#: src/modules/displaymanager/main.py:750
msgid "No display managers selected for the displaymanager module."
msgstr ""
"Nenhum gestor de exibição selecionado para o módulo de gestor de exibição."

#: src/modules/displaymanager/main.py:751
msgid ""
"The displaymanagers list is empty or undefined in bothglobalstorage and "
"displaymanager.conf."
msgstr ""
"A lista de gestores de exibição está vazia ou indefinida no globalstorage e "
"no displaymanager.conf."

#: src/modules/displaymanager/main.py:831
msgid "Display manager configuration was incomplete"
msgstr "A configuração do gestor de exibição estava incompleta"

#: src/modules/initcpiocfg/main.py:36
msgid "Configuring mkinitcpio."
msgstr "A configurar o mkintcpio."

#: src/modules/initcpiocfg/main.py:192
#: src/modules/luksopenswaphookcfg/main.py:100
#: src/modules/machineid/main.py:50 src/modules/initramfscfg/main.py:99
#: src/modules/openrcdmcryptcfg/main.py:83 src/modules/fstab/main.py:317
#: src/modules/localecfg/main.py:145 src/modules/networkcfg/main.py:49
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/initcpio/main.py:33
msgid "Creating initramfs with mkinitcpio."
msgstr ""

#: src/modules/initcpio/main.py:47
msgid "Process Failed"
msgstr ""

#: src/modules/initcpio/main.py:48
msgid ""
"Process <pre>mkinitcpio</pre> failed with error code {!s}. The command was "
"<pre>{!s}</pre>."
msgstr ""

#: src/modules/luksopenswaphookcfg/main.py:35
msgid "Configuring encrypted swap."
msgstr ""

#: src/modules/rawfs/main.py:35
msgid "Installing data."
msgstr "A instalar dados."

#: src/modules/services-openrc/main.py:38
msgid "Configure OpenRC services"
msgstr "Configurar serviços OpenRC"

#: src/modules/services-openrc/main.py:66
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr ""
"Não é possível adicionar o serviço {name!s} ao nível de execução {level!s}."

#: src/modules/services-openrc/main.py:68
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr ""
"Não é possível remover o serviço {name!s} do nível de execução {level!s}."

#: src/modules/services-openrc/main.py:70
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"Serviço de ação desconhecido <code>{arg!s}</code> para serviço {name!s} em "
"nível de execução {level!s}."

#: src/modules/services-openrc/main.py:103
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"<code>rc-update {arg!s}</code> chamar pelo chroot retornou com código de "
"erro {num!s}."

#: src/modules/services-openrc/main.py:110
msgid "Target runlevel does not exist"
msgstr "O nível de execução do destino não existe"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"O caminho para o nível de execução {level!s} é <code>{path!s}</code>, que "
"não existe."

#: src/modules/services-openrc/main.py:119
msgid "Target service does not exist"
msgstr "O serviço do destino não existe"

#: src/modules/services-openrc/main.py:120
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"O caminho para o serviço {name!s} é <code>{path!s}</code>, que não existe."

#: src/modules/plymouthcfg/main.py:36
msgid "Configure Plymouth theme"
msgstr "Configurar tema do Plymouth"

#: src/modules/machineid/main.py:36
msgid "Generate machine-id."
msgstr "Gerar id-máquina"

#: src/modules/packages/main.py:62
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "A processar pacotes (%(count)d / %(total)d)"

#: src/modules/packages/main.py:64 src/modules/packages/main.py:74
msgid "Install packages."
msgstr "Instalar pacotes."

#: src/modules/packages/main.py:67
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "A instalar um pacote."
msgstr[1] "A instalar %(num)d pacotes."

#: src/modules/packages/main.py:70
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "A remover um pacote."
msgstr[1] "A remover %(num)d pacotes."

#: src/modules/bootloader/main.py:51
msgid "Install bootloader."
msgstr ""

#: src/modules/removeuser/main.py:34
msgid "Remove live user from target system"
msgstr "Remover utilizador ativo do sistema de destino"

#: src/modules/initramfs/main.py:35
msgid "Creating initramfs."
msgstr ""

#: src/modules/initramfs/main.py:49
msgid "Failed to run update-initramfs on the target"
msgstr ""

#: src/modules/initramfs/main.py:50 src/modules/dracut/main.py:59
msgid "The exit code was {}"
msgstr ""

#: src/modules/hwclock/main.py:35
msgid "Setting hardware clock."
msgstr ""

#: src/modules/dracut/main.py:36
msgid "Creating initramfs with dracut."
msgstr ""

#: src/modules/dracut/main.py:58
msgid "Failed to run dracut on the target"
msgstr ""

#: src/modules/initramfscfg/main.py:41
msgid "Configuring initramfs."
msgstr "A configurar o initramfs."

#: src/modules/openrcdmcryptcfg/main.py:34
msgid "Configuring OpenRC dmcrypt service."
msgstr ""

#: src/modules/luksbootkeyfile/main.py:35
msgid "Configuring LUKS key file."
msgstr ""

#: src/modules/luksbootkeyfile/main.py:74
msgid "Encrypted rootfs setup error"
msgstr ""

#: src/modules/luksbootkeyfile/main.py:75
msgid "Rootfs partition {!s} is LUKS but no passphrase found."
msgstr ""

#: src/modules/fstab/main.py:38
msgid "Writing fstab."
msgstr ""

#: src/modules/dummypython/main.py:44
msgid "Dummy python job."
msgstr "Tarefa Dummy python."

#: src/modules/dummypython/main.py:97
msgid "Dummy python step {}"
msgstr "Passo Dummy python {}"

#: src/modules/localecfg/main.py:39
msgid "Configuring locales."
msgstr ""

#: src/modules/networkcfg/main.py:37
msgid "Saving network configuration."
msgstr ""
